### PR TITLE
FEATURE: Farmer Greetings Message UI Improvement - Aditi Phatak

### LINKED ISSUE #7: FEATURE: Farmer greetings message UI improvement
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

user

### I can...

not be overwhelmed by the large font of the greeting text and also appreciate the artistic design of the message

### So that...

I can have an improved user experience

### Discussion

1. Having greeting messages in home page and common room page is a great idea in general, but currently the font size is too big and it's just a plain text with black background color.
2. Make the Greeting text slightly smaller, and maybe add movement, hover effect, animation, or gradient background color to make it more stand out.
3. The design is entirely up to the students.

### Acceptance Criteria

- [x] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [x] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

### Implementation Todos

Frontend:
- [x] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [x] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

Testing:
- [x] Frontend: greeting text should have smaller font size and more sophisticated css effects, and students should write unit tests check those attributes.

### PR DESCRIPTION
**Pull request by aditiphatak**
## Overview
In this PR, I edited the Home Page and Play Page such that the greeting text has smaller font size than before as well as more sophisticated CSS effects.

## Screenshots (Optional)
Screenshot 2023-11-27 at 11 12 54 PM

Screenshot 2023-11-27 at 11 12 44 PM

## Feedback Request (Optional)
Please give suggestions on the following:
- This method might not be the best way to do things. What do you think?
- The way this is implemented is the best I can think of. Is it acceptable?
- Where should these things go? I put them here, but it feels out of place.

## Future Possibilities (Optional)
More animation (such as fade-in and fade-out text) 

## Validation (Optional)
1. Download this branch.
2. Run the project.
3. Go to this page.
4. Test this function.

## Tests
- [x] Backend Unit tests (`mvn test`) pass
- [x] Backend Test coverage (`mvn test jacoco:report`) 100%
- [x] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #7

### CODE REVIEW

File: frontend/src/main/components/Commons/CommonsPlay.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    background-image: radial-gradient(circle, #a3c3f0 0%, #64aa9e 100%); 
+    width: 100%;
+    height: 30vh;
+    justify-content: center;
+    display: flex;
+    color: #112731;
+    align-items: center;
+}
+
+/* .fade-in-text {
```
Comment (pconrad) [inline frontend/src/main/components/Commons/CommonsPlay.css]: Why is there commented out code?

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by pconrad — CHANGES_REQUESTED** [review]
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/main/components/Commons/CommonsPlay.css (hunk 1 above)
Comment (aditiphatak) [reply frontend/src/main/components/Commons/CommonsPlay.css]: I have fixed this and deleted the commented out code. Thank you.

**Review by ajs808 — APPROVED** [review]
tested locally, lgtm!

**Review by briggsrrr — COMMENTED** [review]
.

**Review by briggsrrr — APPROVED** [review]
LGTM 🚀!

**Pull request merged**